src/main/java/spark/route/RouteOverview.java

```diff
@@ -0,0 +1,114 @@
+package spark.route;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import spark.FilterImpl;
+import spark.Request;
+import spark.Response;
+import spark.RouteImpl;
+import sun.reflect.ConstantPool;
+
+import static java.util.Collections.singletonList;
+import static spark.Spark.get;
+
+public class RouteOverview {
+
+    /**
+     * Enables a route overview (showing all the mapped routes)
```

Comment:
perhaps explain that this will actually start a Spark server if no previous routes has been added?